File: sync/src/main/java/com/inceptionnotes/sync/events/SimpleMessageServerEvent.java
package com.inceptionnotes.sync.events;

import com.inceptionnotes.sync.Client;

/**
 * Created by jacob on 1/30/18.
 */

public class SimpleMessageServerEvent extends Event {

    public String message;

    public SimpleMessageServerEvent() {
    }

    public SimpleMessageServerEvent(String message) {
        this.message = message;
    }

    @Override
    public void got(Client client) {
        // Do nothing
    }
}

File: sync/src/main/java/com/inceptionnotes/sync/events/SimpleMessageServerEvent.kt
package com.inceptionnotes.sync.events

import com.inceptionnotes.sync.world.Client

/**
 * Created by jacob on 1/30/18.
 */

class SimpleMessageServerEvent : Event {

    var message: String? = null

    constructor()

    constructor(message: String) {
        this.message = message
    }

    override fun got(client: Client) {
        // Do nothing
    }
}
